net-analyzer/hexinject/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>kacper@kolodziej.in</email>
+		<name>Kacper Kołodziej</name>
+	</maintainer>
+	<longdescription lang="en">
```

Comment:
Add. All bugs will still be assigned to you, and proxy-maint will be CC-ed to help you and take action when you go MIA ;-).